addons/getLoadoutsFromUnits/functions/fnc_addMedicItems.sqf

```diff
@@ -0,0 +1,106 @@
+params ["_unit", "_type"];
+
+private _log = [];
+private _items = [];
+private _loadout = getUnitLoadout _unit;
+
+{
+   if (!((_loadout select _x) isEqualTo []) && !((_loadout select _x select 1) isEqualTo [])) then {
+      _items append (_loadout select _x select 1);
+   };
+}forEach [3,4,5];
```

Comment:
missing space

Reply:
Thats an ACE thing, here I can do it as I see fit :P